.github/workflows/push.yml

```diff
@@ -61,8 +61,13 @@ jobs:
       - name: Install hatch
         run: pip install hatch==$HATCH_VERSION
 
-      - name: Verify linting
-        run: hatch run verify
+      - name: Reformat code
```

Comment:
Why do we need this here?

Reply:
Following the same flow as [ucx](https://github.com/databrickslabs/ucx/blob/main/.github/workflows/push.yml)